docs/manual.rst

```diff
@@ -479,11 +479,20 @@ The file format is a bencoded dictionary containing the following fields:
 |                          | for, in case it is a v2 or v1+v2-hybrid torrent. This is a   |
 |                          | 32 byte SHA-256 hash of the info section of the torrent.     |
 +--------------------------+--------------------------------------------------------------+
-| ``pieces``               | A string with piece flags, one character per piece.          |
+| ``pieces``               | **deprecated**. This field is no longer written, but can     |
+|                          | can still be read.                                           |
+|                          | A string with piece flags, one character per piece.          |
 |                          | Bit 1 means we have that piece.                              |
 |                          | Bit 2 means we have verified that this piece is correct.     |
 |                          | This only applies when the torrent is in seed_mode.          |
 +--------------------------+--------------------------------------------------------------+
+| ``pieces2``              | A string where each bit indicates whether we have the piece  |
```

Comment:
Wouldn't it be better to add format version and re-use existing names? Maybe you can even use libtorrent version for such a purpose.

Reply:
that's a good point. I already have a version field. I should use that.
I intend to back-port support for loading the new format, so libtorrent version wouldn't work.